pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -339,6 +339,53 @@ func TestEtcdUpdate(t *testing.T) {
 
 }
 
+func TestNoOpUpdates(t *testing.T) {
+	server, registry := NewTestGenericEtcdRegistry(t)
```

Comment:
Hmm - it seems that there is no easy way to store object with ResourceVersion set in etcd after my fixes.
I manually checked that it works as expected and added a TODO for it.